.circleci/config.yml

```diff
@@ -40,7 +40,6 @@ step-library:
       run:
         name: Prepare
         command: |
-#         if [ $(carthage outdated | grep -cF "latest Carthage version") -eq 0 ]; then brew update && brew upgrade carthage || true; fi
```

Comment:
This commit and the previous one should be rewritten out of history.

/ref https://github.com/mapbox/mapbox-navigation-ios/pull/2404#discussion_r442278601

Reply:
👍